blocks/media-upload-button/index.js

```diff
@@ -78,11 +92,19 @@ class MediaUploadButton extends Component {
 		}
 
 		if ( gallery ) {
```

Comment:
Not exactly, because you _could_ use a media upload button to insert multiple images that are not in a gallery. I would like to see us handle props differently for galleries than passing a `gallery` prop (see: #1979) but was going to leave that as a separate issue.